Package.swift

```diff
@@ -586,9 +586,12 @@ let package = Package(
         ),
         .target(
             name: "TuistDependencies",
-            dependencies: ["TuistCore",
-                           "TuistGraph",
-                           "TuistSupport"]
+            dependencies: [
+                "ProjectDescription",
```

Comment:
Im not sure if `TuistDependencies` should have a dependency to `ProjectDescription`. 🤔 CC: @fortmarek

Reply:
`TuistLoader` also imports `ProjectDescription` and we need similar functionality, so this should be fine.